### PR TITLE
Check bazel version

### PR DESCRIPTION
**Pull request by ash2k**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -87,7 +87,6 @@ run: fmt update-bazel
 	KUBE_PATCH_CONVERSION_DETECTOR=true \
 	KUBE_CACHE_MUTATION_DETECTOR=true \
 	bazel run //cmd/smith:smith_race \
-		--direct_run \
```
Comment (ash2k) [inline Makefile]: `direct_run` is the default behavior starting from `0.14.0` and is deprecated in `0.15.0`

File: WORKSPACE [hunk 2]
```diff
@@ -16,25 +16,36 @@ http_archive(
...
 )
 
 http_archive(
     name = "com_github_atlassian_bazel_tools",
-    sha256 = "c6abcc19e65707a0232e4523d68c0d320f780db9f3eb0e36adea4fcd1055463b",
-    strip_prefix = "bazel-tools-06d357a7c08ab0854821e106f0891aa80d130b35",
-    urls = ["https://github.com/atlassian/bazel-tools/archive/06d357a7c08ab0854821e106f0891aa80d130b35.tar.gz"],
+    sha256 = "016af5647946422b9314d1df988aa7bc48ac7c1d1bf5e578ade8cf08e0ecfffe",
+    strip_prefix = "bazel-tools-37f8cb882a0226bb1fd7baf84f70b826e14e9469",
+    urls = ["https://github.com/atlassian/bazel-tools/archive/37f8cb882a0226bb1fd7baf84f70b826e14e9469.tar.gz"],
```
Comment (ash2k) [inline WORKSPACE]: This picks up the fixed goimports version

**Comment by nilebox** [comment]
LGTM, but the build fails

**Comment by ash2k** [comment]
Build fails because we need newer bazel in Travis :)
```
(10:34:40) ERROR: /home/travis/gopath/src/github.com/atlassian/smith/WORKSPACE:47:1: Traceback (most recent call last):
	File "/home/travis/gopath/src/github.com/atlassian/smith/WORKSPACE", line 47
		versions.check(minimum_bazel_version = "0.14.0")
	File "/home/travis/.cache/bazel/_bazel_travis/21b3068201190d7334091ffc6389815b/external/bazel_skylib/lib/versions.bzl", line 104, in versions.check
		fail("\nCurrent Bazel version is {}, ...))
Current Bazel version is 0.13.1, expected at least 0.14.0
```

**Pull request merged**